### DISCUSSION TITLE
What is the equivalent of Properties.Settings with Avalonia?

### DISCUSSION THREAD

**Discussion by zonetecde** [AvaloniaUI/Avalonia#12129] [Q&A] [answered]
Hello,

In a WPF application project, you can store variables that the application will keep somewhere on the disk in order to have them again the next time it is opened.

Is there an equivalent to this with Avalonia? I have a multi-platform project (web, android and desktop)

**Comment by timunie** [comment, marked as answer]
There is no single way that works for x-plat scenarios. If you are just using Desktop, you can for example use: https://docs.avaloniaui.net/docs/next/concepts/reactiveui/data-persistence 

for x-plat, you may need to write a service which you pull in per platform. See: https://github.com/AvaloniaUI/Avalonia/discussions/7190

**Reply by zonetecde** [reply]
thank you, I'll look into that

**Comment by maxkatz6** [comment]
https://learn.microsoft.com/en-us/dotnet/core/extensions/configuration
https://learn.microsoft.com/en-us/visualstudio/ide/managing-application-settings-dotnet?view=vs-2022
